### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by keliphan**
Added redux files for homepage, signup, query, and plantshelf pages. Styling and routing included.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (simple-sifu) [inline .vscode/settings.json:3]: can you remove this vscode file?

File: app-pages/homepage/styles.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
+/* document styling */
+body {
```
Comment (lpham598) [inline app-pages/homepage/styles.css:2]: Please remove these extra pages since they aren't needed anymore.

File: app-pages/homepage/index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (lpham598) [inline app-pages/homepage/index.html:5]: Please remove these extra pages since they aren't needed anymore.

File: app-pages/plantshelf/index.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>PLANT SHELF</title>
+    <link rel="stylesheet" href="./styles.css" />
+  <body>
+    <div class="container">
```
Comment (lpham598) [inline app-pages/plantshelf/index.html:9]: Read comments above.

File: app-pages/plantshelf/styles.css [hunk 5]
```diff
@@ -0,0 +1,101 @@
+body {
+  font-family: courier;
+  background-color: rgb(132, 187, 139);
+  margin: 0
+}
+
+img {
```
Comment (lpham598) [inline app-pages/plantshelf/styles.css:7]: Read comments above.

File: .vscode/settings.json (hunk 1 above)
Comment (keliphan) [reply .vscode/settings.json]: removed, thanks!

File: app-pages/homepage/styles.css (hunk 2 above)
Comment (keliphan) [reply app-pages/homepage/styles.css]: removed, thanks!

File: app-pages/homepage/index.html (hunk 3 above)
Comment (keliphan) [reply app-pages/homepage/index.html]: removed, thanks!

File: app-pages/plantshelf/index.html (hunk 4 above)
Comment (keliphan) [reply app-pages/plantshelf/index.html]: removed, thanks!

File: app-pages/plantshelf/styles.css (hunk 5 above)
Comment (keliphan) [reply app-pages/plantshelf/styles.css]: removed, thanks!

**Pull request merged**